scripts/update-requirements

```diff
@@ -0,0 +1,133 @@
+    add_sha256sums(req_path)
+    add_sha256sums(reqdev_path)
+
+
+def add_sha256sums(path: str) -> None:
+    """Adds all the required sha256sums to the wheels
+    
+    :param path: path to the requirements file.
+    :type path: str
+    :return: None
+    :rtype: None
+    """
+    files = []
+
+    with open("sha256sums.txt") as fobj:
+        lines = fobj.readlines()
+
+    for line in lines:
+        line = line.strip()
+        # Skip any sources
+        if line.find(".whl") == -1:
+            continue
+        files.append(line.split())
+
+    with open(path) as fobj:
+        lines = fobj.readlines()
+
+    newlines = []
+    missing_wheels = []
+    for line in lines:
+        line = line.strip()
+        words = line.split()
+        values = words[0].split("==")
+        values[1] = values[1].strip(';')
+        packagename = "-".join(values)
+        othername = "-".join([values[0].replace("-", "_"), values[1]])
+
+        for name in files:
+            lowername = name[1].lower()
+            digest = name[0]
+            package_othername = packagename.replace("-", "_")
+            # Now check if a wheel is already available
+            if lowername.startswith(packagename) or lowername.startswith(
+                othername
+            ):
+                # Check if the sha256sum is already there or not
+                if digest not in line:
+                    # Now add the hash to the line
+                    line = "{} --hash=sha256:{}\n".format(line, digest)
+        newlines.append(line)
+        if line.find("--hash") == -1:  # Missing wheel
+            missing_wheels.append(values)
+
+    # Do not update the file if missing wheels
+    if missing_wheels:
+        print("The following dependent wheel(s) are missing:")
+        for value in missing_wheels:
+            print("{0}=={1}".format(value[0], value[1]))
+
+        print("\nPlease build the wheel by using the following command:\n")
+        print("\tPKG_DIR={0} make build-wheels\n".format(os.environ["PKG_DIR"]))
+        print("Then sync the newly built wheels and sources to the s3 bucket.")
+        print("Also update the index HTML files accordingly and sync to s3.")
+        print("After these steps, please rerun the command again.")
+
+        sys.exit(1)
+
+    # Now update the file
+    with open(path, "w") as fobj:
+        for line in newlines:
+            fobj.write(line)
+
+
+def cleanup(path: str) -> None:
+    """Cleans up requirement files
+    
+    :param path: The file to cleanup
+    :type path: str
+    :return: None
+    :rtype: None
+    """
+
+    with open(path) as fobj:
+        lines = fobj.readlines()
+
+    finallines = []
+
+    for line in lines:
+        # Skip the PyPI index line
+        if line.startswith("-i https://pypi.org/simple"):
+            continue
+        # For PyQt5
+        elif line.startswith("pyqt5"):
+            continue
+        # For sip
+        elif line.startswith("sip"):
```

Comment:
why? (needs comment)

Reply:
looks like this requirement is used in dev only, but doesn't hurt to keep in here